device-service

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
Should this be a folder not a file and be under `legacy-design`?
Same question for the other files that should be folders.